### ISSUE TITLE
check for int input can  be done without try except

### ISSUE THREAD

**Issue by blusamurai** [croods701/python_utils#1]
I found that you are using `try except` for checking input which is not considered a good practice.
I think it should be changed with other alternatives such as `str.isdigit()`

**Comment by parul-taley** [comment]
Yeah, Good observation. If you don't mind can you open a PR for this

**Comment by blusamurai** [comment]
Ok no problem, I will do that
